src/Microsoft.ML.Transforms/ProjectionCatalog.cs

```diff
@@ -63,7 +63,7 @@ public static RandomFourierFeaturizingEstimator CreateRandomFourierFeatures(this
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="columns"> Describes the parameters of the lp-normalization process for each column pair.</param>
-        public static LpNormalizingEstimator LpNormalize(this TransformsCatalog.ProjectionTransforms catalog, params LpNormalizingTransformer.LpNormColumnInfo[] columns)
+        public static LpNormalizingEstimator LpNormalize(this TransformsCatalog.ProjectionTransforms catalog, params LpNormalizingEstimator.LpNormColumnInfo[] columns)
```

Comment:
Add a Remarks section explaining what this Estimator does, with external links if necessary.

Reply:
Same here.

---
In reply to: [254013908](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254013908) [](ancestors = 254013908)